Readers/interface/RecoHadTauReader.h

```diff
@@ -12,6 +12,7 @@ class GenLeptonReader;
 class GenHadTauReader;
 class GenJetReader;
 class TauESTool;
+class TriggerInfo;
```

Comment:
Is this necessary? I don't see that `TriggerInfo` object is actually used by this class.